src/components/Disputes/actions/DisputeAppeal.js

```diff
@@ -26,14 +25,7 @@ function DisputeAppeal({ onRequestAppeal, confirm }) {
         {actionLabel}
       </Button>
       <Info mode="description">
-        Anyone can trigger this action, and whoever does will get some{' '}
-        <span
-          css={`
-            color: ${theme.help};
-          `}
-        >
-          rewards.
-        </span>
+        Anyone holding the required appeal deposit can trigger this action.
```

Comment:
```suggestion
        Anyone can deposit the required amount to appeal and trigger this action.
```

I notice "anyone" is also in a `<strong>` in the other section; should it also be the case here?

Reply:
Yes, good catch